Makefile

```diff
@@ -120,11 +120,11 @@ protos: $(PROTO_GOS)
 	protoc -I $(GOPATH)/src:./vendor:./$(@D) --gogoslick_out=plugins=grpc,Mgoogle/protobuf/any.proto=github.com/gogo/protobuf/types,:./$(@D) ./$(patsubst %.pb.go,%.proto,$@)
 
 lint:
-	./tools/lint -notestpackage -novet -ignorespelling queriers -ignorespelling Queriers .
-
 	# -stdmethods=false disables checks for non-standard signatures for methods with familiar names.
 	# This is needed because the Prometheus storage interface requires a non-standard Seek() method.
-	go vet -stdmethods=false ./pkg/...
+	go vet -tags netgo -stdmethods=false ./pkg/...
```

Comment:
I think go vet is on by default. We use it on Loki.

>govet (vet, vetshadow): Vet examines Go source code and reports suspicious constructs, such as Printf calls whose arguments do not align with the format string [fast: true, auto-fix: false]

see https://github.com/golangci/golangci-lint#quick-start

Reply:
I did this test: change a printf format string to invalid, e.g. `%i`.
I get a warning from `go vet` but not from `golangci-lint`.
So I am un-convinced.

Reply:
Interesting I have the same issue in loki but if I ran it for the specific package it works. Tried to clean my cache also using `golangci-lint cache clean`.

Reply:
This is failing `golangci-lint run -v ./pkg/querier/queryrange`

but not 

`golangci-lint run -v ./...`

Reply:
Found the issue on Loki side, one file was producing warning.

```
WARN [runner] Can't run linter goanalysis_metalinter: misspell: can't get file /Users/ctovena/go/src/github.com/grafana/loki/pkg/logql/pkg/logql/expr.y contents: can't read file /Users/ctovena/go/src/github.com/grafana/loki/pkg/logql/pkg/logql/expr.y: open /Users/ctovena/go/src/github.com/grafana/loki/pkg/logql/pkg/logql/expr.y: no such file or directory
```

This was caused by a comment `//line expr.y`, I cleanup the file and it now catches %i in fmt.

Reply:
FYI it was also causing some linter to be skipped.

Reply:
You have the same kind of warning.

```
WARN [runner] Can't run linter goanalysis_metalinter: fact_purity: failed prerequisites: buildssa@github.com/cortexproject/cortex/pkg/chunk [github.com/cortexproject/cortex/pkg/chunk/cache.test] 
```

Not sure why the exit code is not 1.

Reply:
I found various issues relating to this - https://github.com/golangci/golangci-lint/issues/866, https://github.com/golangci/golangci-lint/issues/827